configs/config.d/forward.conf

```diff
@@ -0,0 +1,6 @@
+<source>
+  @type forward
+  port 24224
```

Comment:
I would at least add a "# Default." comment to this line. The problem is that other tools will attempt to connect to this port, so it makes sense to communicate to the user that changing this not advisable.